### PR TITLE
draft docs for existing intents

### PR DESCRIPTION
**Pull request by tyanas**

### CODE REVIEW

File: ai.image.ocr.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    ...
+]
+```
+
+## Getting information about a provider
+
+To get information about a provider with a given ID, send a GET request to `https://api.inten.to/ai/image/ocr/PROVIDER_ID`.
+
+```sh
+curl -H 'apikey: YOUR_INTENTO_KEY' 'https://api.inten.to/ai/image/ocr/ai.image.ocr.google.cloud_vision_annotate_api.1-0'
```
Comment (dimavin) [inline ai.image.ocr.md]:
```suggestion
curl -H 'apikey: YOUR_INTENTO_KEY' 'https://api.inten.to/ai/image/ocr/ai.image.ocr.google.cloud_vision_api.1-1'
```

File: ai.image.ocr.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+```
+
+The response contains a list of the metadata fields and values available for the provider:
+
+```json
+{
+    "id": "ai.image.ocr.google.cloud_vision_annotate_api.1-0",
```
Comment (dimavin) [inline ai.image.ocr.md]:
```suggestion
    "id": "ai.image.ocr.google.cloud_vision_api.1-1",
```

Comment (dimavin) [inline ai.image.ocr.md]: or more broadly:

> {
  "id": "ai.image.ocr.google.cloud_vision_api.1-1",
  "vendor": "Google Cloud",
  "logo": "https://inten.to/static/img/api/ggl_vision.png",
  "billing": true,
  "description": "Vision API",
  "production": true,
  "integrated": false,
  "billable": true,
  "own_auth": true,
  "stock_model": true,
  "custom_model": false,
  "auth": {
    "key": "YOUR_KEY"
  }
}

File: ai.image.tagging.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
+# ai.image.tagging
+
+Image tagging generates a list of words, or tags, that are relevant to the content of the supplied image.
```
Comment (dimavin) [inline ai.image.tagging.md]:
```suggestion
Image tagging generates a list of words or tags that are relevant to the content of the supplied image.
```

File: ai.speech.transcribe.md [hunk 4]
```diff
@@ -0,0 +1,220 @@
+# ai.speech.transcribe
+
+Text content classification.
```
Comment (dimavin) [inline ai.speech.transcribe.md]:
```suggestion
Transcribe a speech record into text.
```

File: ai.text.transliterate.md [hunk 5]
```diff
@@ -0,0 +1,219 @@
+# ai.text.transliterate
+
+Converts text in one language from one script to another script.
```
Comment (dimavin) [inline ai.text.transliterate.md]:
```suggestion
Converts a text from one script to another script (e.g. katakana to latin)
```

File: ai.image.ocr.md [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+    "context": {
+        "source": "..."
+    },
+    "service": {
+        "provider": "ai.image.ocr.abbyy.cloud_ocr"
+    }
+}'
+```
+
+The response contains the speech transcribe results:
```
Comment (tyanas) [inline ai.image.ocr.md]:
```suggestion
The response contains the results:
```

**Pull request merged**